common/sagas/swap/rates.ts

```diff
@@ -1,11 +1,19 @@
 import { showNotification } from 'actions/notifications';
-import { loadBityRatesSucceededSwap } from 'actions/swap';
+import {
+  loadBityRatesSucceededSwap,
+  loadShapeshiftRatesSucceededSwap,
+  changeSwapProvider,
+  ChangeProviderSwapAcion
+} from 'actions/swap';
 import { TypeKeys } from 'actions/swap/constants';
 import { getAllRates } from 'api/bity';
 import { delay, SagaIterator } from 'redux-saga';
-import { call, cancel, fork, put, take, takeLatest } from 'redux-saga/effects';
+import { call, select, cancel, fork, put, take, takeLatest, race } from 'redux-saga/effects';
+import shapeshift from 'api/shapeshift';
+import { getSwap } from 'sagas/swap/orders';
 
 const POLLING_CYCLE = 30000;
+const SHAPESHIFT_TIMEOUT = 10000;
```

Comment:
Should we consider changing to a higher value?

Reply:
If we want to get especially fancy, we could attempt to determine if the device is mobile / desktop, and increase the timeout on mobile.

For the MVP, I think 10s is fine.